### PR TITLE
Add continue_on_error option to continue on error while crawling

### PR DESCRIPTION
**Pull request by kneubi**
If you need to scan large directories you might hit a Permission denied exception from time to time, which might be o.k for you. I have added a continue_on_error fs option, which let you decide if you want to continue anyway and just print a warning instead of completely stop scanning.
If we need to add a Testcase, i need probably some help again.

### CODE REVIEW

File: src/main/java/fr/pilato/elasticsearch/crawler/fs/FsCrawlerImpl.java [hunk 1]
```diff
@@ -382,8 +382,16 @@ private void addFilesRecursively(FileAbstractor<?> path, String filepath, LocalD
...
                             if (child.lastModifiedDate.isAfter(lastScanDate) ||
                                     (child.creationDate != null && child.creationDate.isAfter(lastScanDate))) {
-                                indexFile(child, stats, filepath, path.getInputStream(child), child.size);
-                                stats.addFile();
+                                try {
+                                    indexFile(child, stats, filepath, path.getInputStream(child), child.size);
+                                    stats.addFile();
+                                } catch (java.io.FileNotFoundException e) {
+                                    if (fsSettings.getFs().isContinueOnError()) {
+                                        logger.warn("Unable to open Input Stream for {}, skipping...",e.getMessage());
```
Comment (dadoonet) [inline src/main/java/fr/pilato/elasticsearch/crawler/fs/FsCrawlerImpl.java]: Just add a space before `e.getMessage());` please.

**Comment by dadoonet** [comment]
BTW I'd recommend that you work everytime on another branch than your master branch.

**Comment by kneubi** [comment]
I have added the the white space. And yes i will begin to use feature branches from now on, since i have some more ideas.

**Comment by dadoonet** [comment]
Merged. Thanks!

**Pull request merged**